management.py

```diff
@@ -0,0 +1,19 @@
+from model import Model
+
+INITIAL_MONEY = 1000
+SPENDING_PROB = 0.5 
+SPENDING_AMOUNT = 100
+NUM_PEOPLE = 10
+SALARY = 1000
+NUM_STEPS = 10
+
+model = Model(NUM_PEOPLE, INITIAL_MONEY, 
+                 SPENDING_PROB, SPENDING_AMOUNT,
+                 SALARY)
+
+for i in range(NUM_STEPS):
+    model.step()
+
+agent_money = model.datacollector.get_agent_vars_dataframe()
+
+print(agent_money.xs(9, level="Step")["Money"])
```

Comment:
I would move lines 14-17 to a run() method of the Model() class with a parameter on the number of steps, with it returning the dataframe output. That is, create a method run(num_steps=10) in Model().

For a user, their workflow would then be
`model = Model()`
`df = model.run(10)`
where df is the output data frame. This workflow would work well in a Jupyter notebook, and they could then analyze the dataframe there or save it to file, etc. With the possibility of run()'ing some more.